.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I am not sure we need this. We should encourage people to use the forums (discord / zulip / u.r.l.o / Reddit) instead for faster chances of solution.

Reply:
Does clap have a dedictated forum?

Reply:
Or a chat?

Reply:
There is gitter, but it's pretty sparse.